src/microbe_stage/SpawnSystem.cs

```diff
@@ -259,23 +257,18 @@ private void SpawnEntities(Vector3 playerPosition, int existing, int spawnsLeftT
                     will fail the second condition, so entities still only
                     spawn within the spawning region.
                     */
-                    float distanceX = (float)random.NextDouble() * spawnType.SpawnRadius -
-                        (float)random.NextDouble() * spawnType.SpawnRadius;
-                    float distanceZ = (float)random.NextDouble() * spawnType.SpawnRadius -
-                        (float)random.NextDouble() * spawnType.SpawnRadius;
+                    float displacementDistance = (float)random.NextDouble() * spawnType.SpawnRadius;
```

Comment:
I think I added an extension method called NextFloat use that instead of NextDouble here.